src/game_objects/player.cpp

```diff
@@ -113,15 +123,15 @@ void Player::start_walk() {
 
 // Server collision
 void Player::s_on_collision(GameObject *other) {
-    // TODO: actual game logic here...
-
-    // Then notify clients that this collision happened.
-    events::dungeon::network_collision_event(id, other->get_id());
+    // By default, does not need to notify the client of any collisions.
+    // If it triggers a trap, the trap will notify the client that the
+    // player is hit. 
 }
 
 // Graphical collision
 void Player::c_on_collision(GameObject *other) {
-    transform.rotate(0, 0.1f, 0);
+    if (dynamic_cast<Spikes*>(other))
```

Comment:
@richard1 and I think that this `take_damage` call should go in `Spikes`, to prevent `Player` from bloating up.